frappe/public/scss/common/grid.scss

```diff
@@ -150,6 +150,7 @@
 .grid-row > .row {
 	.col:last-child {
 		border-right: none;
+		min-width: 50px;
```

Comment:
```suggestion
		min-width: 0;
```
And instead of using `hidden-xs` use `hidden-md` it should fix it properly

https://github.com/frappe/frappe/assets/30859809/67c3d683-0b43-4427-b956-ba36411953bc

Reply:
Yes, i tried this approach but on small viewport it has less padding so i chose 50px and also i thought it would be ideal to show "edit" text on md size.

### Look at Edit Column
https://github.com/frappe/frappe/assets/39730881/63a7b1e6-7117-4b25-9ea2-ae8be16ae53a

Reply:
It is fine to not show Edit in a smaller viewport, the less padding doesn't look good. Also using 50px hardcoded value is not a nice practice